core/utils/htable_bench.cc

```diff
@@ -119,7 +138,7 @@ BENCHMARK_DEFINE_F(HTableFixture, BessGet)(benchmark::State &state) {
 
 BENCHMARK_REGISTER_F(HTableFixture, BessGet)
     ->RangeMultiplier(4)
-    ->Range(4, 4 << 20);
+    ->Range(4, 4 << 18);
```

Comment:
Let's make it back to 4 << 20. 16M could be a meaningful number since often people test FIB scalability with 16M entries...